amplify/src/main/java/com/github/stkent/amplify/views/QuestionView.java

```diff
@@ -26,11 +25,11 @@
     View getView();
 
     @NonNull
-    Button getPositiveButton();
+    View getPositiveButton();
 
     @NonNull
-    Button getNegativeButton();
+    View getNegativeButton();
 
-    void setQuestion(@NonNull final Question question);
+    void setQuestion(@NonNull final String question);
```

Comment:
Hmm not sure, it's still just the question string. Right now there isn't a title, maybe we will add it tomorrow? We can hold off on this PR till we can look at this in person.